docs/sources/administration/data-source-management/team_ldap.md

```diff
@@ -0,0 +1,130 @@
+---
+aliases:
+  - ../datasources/datasource_permissions/team_ldap
+  - ../datasources/datasource_permissions/teamldap
+  - ../enterprise/datasource_permissions/team_ldap
+  - ../enterprise/datasource_permissions/teamldap
+  - ../permissions/datasource_permissions/team_ldap
+  - ../permissions/datasource_permissions/teamldap
+description: Data source permissions for Teams for Grafana administrators
+labels:
+  products:
+    - enterprise
+    - cloud
+title: Data source permissions for Team LDAP
+weight: 100
+---
+
+# Team LBAC
+
+Grafana's new Team LBAC (Label-Based Access Control) feature for Loki is a significant enhancement that simplifies and streamlines data source access management based on team memberships. Users wanting fine grained access to data sources such as Loki with X amount of teams with different levels of access can make use of Team LBAC.
+
+This feature addresses a common challenge faced by Grafana users: managing Loki data source access for different teams. Previously, this led to the creation of numerous connections and duplicate dashboards, hampering scalability and user experience. With Team LBAC, users can now configure access to specific labels based solely on team memberships.
+
+## Team LBAC rules
+
+Users who want teams with a specific set of label selectors can add rules for each team. Team LBAC rules have no upper limit as to how many you can configure per team.
+
+Team LBAC rules are added to the http request to Loki data source. Setting up Team LBAC rules for any team will apply those rules to the teams.
+
+**Note:** Any teams within Grafana without a rule will be able to query all logs if there are role based queriying setup. See <> for more information.
+
+Configuring multiple rules for a team acts the same way as CAP rules, each rule is evaluated separately. If a team has two rules configured for it, both rules will be applied to the request and the result will be the [logical disjunction](https://en.wikipedia.org/wiki/Logical_disjunction), a "OR" operation of the two rules.
```

Comment:
I thinks it's better to explain what is CAP

Reply:
I think OR is enough to explain result, "logical disjunction" sounds terrifying :)

Reply:
> I think OR is enough to explain result, "logical disjunction" sounds terrifying :)

The `logical disjunction` monster is coming for you! 😈 

il remove it